rats-tls/cmake/SGXSSL.cmake

```diff
@@ -7,36 +7,37 @@ include(ExternalProject)
 include(FindSGXSSL)
 
 # Patch
-set(_patch_script "${CMAKE_CURRENT_SOURCE_DIR}/patch_script.sh")
+set(SGX_SSL "${CMAKE_CURRENT_SOURCE_DIR}/src/external/sgx-ssl")
```

Comment:
Could you change the variable name from "SGX_SSL" to ""SGX_SSL_PATCH_DIR? 
That will be more clearly what purpose of the variable is for.